src/python/WMCore/WMExceptions.py

```diff
@@ -163,6 +163,7 @@
                       60307: "General failure during files stage out.",  # (WMA, CRAB3)
                       60311: "Local Stage Out Failure using site specific plugin.",  # (WMA, CRAB3)
                       60312: "Failure in staging in log files during log collection (WMAgent).",  # (WMA)
+                      60313: "Failed to clean up every single no longer needed files (WMAgent).",  # (WMA)
```

Comment:
I don't like this error message, but my brain doesn't come up with a better one. Perhaps @todor-ivanov or @goughes have a better idea? I simply want to say that the Cleanup job did not succeed to delete even a single file :-)

Reply:
So there were a bunch of files to be deleted and 0 files were deleted?

"Failed to clean up any files that were no longer needed (WMAgent)."

Reply:
Sorry Erik, it looks like I missed your comment. Yes, you got it right. I'm updating this message! Thanks